client/api/UpdateProcessor.cs

```diff
@@ -71,16 +71,11 @@ public void OnStreamConnected()
             this.callback.OnStreamConnected();
         }
 
-        private async Task StartAfterInterval()
-        {
-            await Task.Delay(TimeSpan.FromSeconds(this.config.pollIntervalInSeconds));
-            Start();
-        }
+
         public void OnStreamDisconnected()
         {
             this.callback.OnStreamDisconnected();
-            Stop();
-            _ = StartAfterInterval();
```

Comment:
The `EventSource` class now manages reconnection itself - we don't attempt to restart it in this class.